### DISCUSSION TITLE
Welcome to stable-pi-core Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/stable-pi-core#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pigurus** [comment]


**Reply by Sipneo** [reply]
Price 314,159 пи сетевая утопия 😱

**Comment by Mackenzie-617** [comment]
$314,159 or $314.159?

**Reply by Sipneo** [reply]
thousand

**Comment by Utsavdhamecha** [comment]
Bro when this things are impliment?
